<ui version="4.0" >
 <class>contactListSettingsClass</class>
 <widget class="QWidget" name="contactListSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>contactListSettings</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>464</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Main</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../../../../qutim.qrc" >
        <normaloff>:/icons/crystal_project/contactlist.png</normaloff>:/icons/crystal_project/contactlist.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QCheckBox" name="showGroupsBox" >
           <property name="text" >
            <string>Show groups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="hideEmptyBox" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="text" >
            <string>Hide empty groups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="offlineBox" >
           <property name="text" >
            <string>Hide offline users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="separatorBox" >
           <property name="text" >
            <string>Hide online/offline separators</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="nilBox" >
           <property name="text" >
            <string>Clear 'Not in list' on startup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="authorizedBox" >
           <property name="text" >
            <string>Don't underline "Authorization required" users</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="birthdayBox" >
           <property name="text" >
            <string>Don't mark out users' birthdays</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="expandBox" >
           <property name="text" >
            <string>Expand account list on startup</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>446</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>464</width>
        <height>267</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Customize</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../../../../qutim.qrc" >
        <normaloff>:/icons/crystal_project/fonts.png</normaloff>:/icons/crystal_project/fonts.png</iconset>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QGroupBox" name="customBox" >
         <property name="title" >
          <string>Customize font:</string>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <layout class="QHBoxLayout" >
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <widget class="QCheckBox" name="accountFontBox" >
                <property name="text" >
                 <string>Account:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="groupFontBox" >
                <property name="text" >
                 <string>Group:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="onlineFontBox" >
                <property name="text" >
                 <string>Online:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="offlineFontBox" >
                <property name="text" >
                 <string>Offline:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" >
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QFontComboBox" name="accountFontComboBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="accountFontSizeBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimum" >
                   <number>6</number>
                  </property>
                  <property name="maximum" >
                   <number>24</number>
                  </property>
                  <property name="value" >
                   <number>9</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="accountColorButton" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="icon" >
                   <iconset resource="../../../../qutim.qrc" >
                    <normaloff>:/icons/crystal_project/colorpicker.png</normaloff>:/icons/crystal_project/colorpicker.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QFontComboBox" name="groupFontComboBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="groupFontSizeBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimum" >
                   <number>6</number>
                  </property>
                  <property name="maximum" >
                   <number>24</number>
                  </property>
                  <property name="value" >
                   <number>6</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="groupColorButton" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="icon" >
                   <iconset resource="../../../../qutim.qrc" >
                    <normaloff>:/icons/crystal_project/colorpicker.png</normaloff>:/icons/crystal_project/colorpicker.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QFontComboBox" name="onlineFontComboBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="onlineFontSizeBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimum" >
                   <number>6</number>
                  </property>
                  <property name="maximum" >
                   <number>24</number>
                  </property>
                  <property name="value" >
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="onlineColorButton" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="icon" >
                   <iconset resource="../../../../qutim.qrc" >
                    <normaloff>:/icons/crystal_project/colorpicker.png</normaloff>:/icons/crystal_project/colorpicker.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" >
                <item>
                 <widget class="QFontComboBox" name="offlineFontComboBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="offlineFontSizeBox" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="minimum" >
                   <number>6</number>
                  </property>
                  <property name="maximum" >
                   <number>24</number>
                  </property>
                  <property name="value" >
                   <number>10</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QToolButton" name="offlineColorButton" >
                  <property name="enabled" >
                   <bool>false</bool>
                  </property>
                  <property name="text" >
                   <string/>
                  </property>
                  <property name="icon" >
                   <iconset resource="../../../../qutim.qrc" >
                    <normaloff>:/icons/crystal_project/colorpicker.png</normaloff>:/icons/crystal_project/colorpicker.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="../../../../qutim.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>accountFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>accountFontComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>accountFontSizeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>accountFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>accountColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>102</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel" >
     <x>451</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupFontComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupFontSizeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>99</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>groupFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>112</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel" >
     <x>451</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlineFontComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>109</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlineFontSizeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>onlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>onlineColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>125</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel" >
     <x>451</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>offlineFontComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>236</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>offlineFontSizeBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>115</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>419</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>offlineFontBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>offlineColorButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>127</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel" >
     <x>451</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showGroupsBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>hideEmptyBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>67</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel" >
     <x>63</x>
     <y>87</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
